cli/conf.py

```diff
@@ -38,102 +35,6 @@ def load_config_file():
         "Please set it in one of the following paths:\n")
 
 
-def IniFileType(value):
```

Comment:
Function names should be lowercase, with words separated by underscores.
(https://www.python.org/dev/peps/pep-0008/#function-names)